controller_costmap:
    global_frame: odom
    robot_base_frame: base_footprint
    update_frequency: 5.0
    publish_frequency: 2.5

    width: 30
    height: 30
    resolution: 0.05
     
    transform_tolerance: 0.5
    rolling_window: true
    always_send_full_costmap: true

    plugins:
      - {name: static_layer,            type: "navit_costmap::StaticLayer"}
      - {name: obstacle_layer,          type: "navit_costmap::VoxelLayer"}
      - {name: inflation_layer,         type: "navit_costmap::InflationLayer"}
    

    obstacle_layer:
        observation_sources: lidar
        lidar:
            topic: base_scan
            sensor_frame: base_laser_link
            data_type: LaserScan
            marking: true
            clearing: true
            obstacle_range: 8
            raytrace_range: 20

    inflation_layer:
        inflation_radius: 0.4
        cost_scaling_factor: 1.5


